src/models/polar/kernels.jl

```diff
@@ -414,3 +414,34 @@ end
     slines[ℓ + nlines] = to_flow
 end
 
+function branch_flow_kernel_zygote(
```

Comment:
I like this vectorized function. Do you know how many allocations does it lead to on the GPU?

Reply:
I will look into this.